diagnostic.md

```diff
@@ -23,7 +27,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('li').parent().children('form')
```

Comment:
Almost.  You need to go up one more level.  `$('li').parent()` selects the direct ancestor of `<li>`, which is `<ol>`.  A few possible ways are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.